docs/src/use/command-line-interface.md

```diff
@@ -67,36 +67,33 @@ You can view all the CLI options by running `npx eslint -h`.
 eslint [options] file.js [file.js] [dir]
 
 Basic configuration:
-  --no-eslintrc                   Disable use of configuration from .eslintrc.*
-  -c, --config path::String       Use this configuration, overriding .eslintrc.* config options if present
-  --env [String]                  Specify environments
-  --ext [String]                  Specify JavaScript file extensions
+  --no-config-lookup              Disable look up for eslint.config.js
+  -c, --config path::String       Use this configuration instead of eslint.config.js, eslint.config.mjs, or
+                                  eslint.config.cjs
+  --inspect-config                Open the config inspector with the current configuration
   --global [String]               Define global variables
   --parser String                 Specify the parser to be used
   --parser-options Object         Specify parser options
-  --resolve-plugins-relative-to path::String  A folder where plugins should be resolved from, CWD by default
 
-Specify rules and plugins:
+Specify Rules and Plugins:
   --plugin [String]               Specify plugins
   --rule Object                   Specify rules
-  --rulesdir [path::String]       Load additional rules from this directory. Deprecated: Use rules from plugins
 
-Fix problems:
+Fix Problems:
   --fix                           Automatically fix problems
   --fix-dry-run                   Automatically fix problems without saving the changes to the file system
   --fix-type Array                Specify the types of fixes to apply (directive, problem, suggestion, layout)
 
-Ignore files:
-  --ignore-path path::String      Specify path of ignore file
```

Comment:
`--ignore-path` still has a paragraph below.

Reply:
Yup, that's because it's still valid in eslintrc mode.